lsp-clients.el

```diff
@@ -828,5 +830,120 @@ responsiveness at the cost of possible stability issues."
                   :priority -1
                   :server-id 'cmakels))
 
+;;; Rf
+(defgroup lsp-rf nil
+  "Settings for Robot Framework Language Server."
+  :group 'lsp-mode
+  :tag "Language Server"
+  :link '(url-link "https://github.com/tomi/vscode-rf-language-server.git"))
+
+(defcustom lsp-rf-language-server-main-file "~/.vscode/extensions/tomiturtiainen.rf-intellisense-2.8.0/server/server.js"
+  "Path to the server.js file of the rf-intellisense server. Accepts a string"
+  :type 'string
+  :group 'lsp-rf)
+
+(defcustom lsp-rf-language-server-include-paths []
+  "An array of files that should be included by the parser. Glob patterns as strings are accepted (eg. *.robot between double quotes)"
+  :type 'lsp-string-vector
+  :group 'lsp-rf)
+
+(defcustom lsp-rf-language-server-exclude-paths []
+  "An array of files that should be ignored by the parser. Glob patterns as strings are accepted (eg. *bad.robot between double quotes)"
+  :type 'lsp-string-vector
+  :group 'lsp-rf)
+
+(defcustom lsp-rf-language-server-python-keywords nil
+  "DEPRECATED. Use rfLanguageServer.includePaths instead"
+  :type 'boolean
+  :group 'lsp-rf)
+
+(defcustom lsp-rf-language-server-dir "~/.vscode/extensions/tomiturtiainen.rf-intellisense-2.8.0/server/library-docs/"
+  "Libraries directory for libraries in lsp-rf-language-server-libraries"
+  :type 'string
+  :group 'lsp-rf)
+
+(defcustom lsp-rf-language-server-libraries '("BuiltIn-3.1.1" "Collections-3.0.4")
+  "Libraries whose keywords are suggested with auto-complete"
```

Comment:
should be vector. Note that emacs27+ json serialization treats list as objects

Reply:
This means that for now it works correctly, but from emacs 27+ on it will fail?

Reply:
yes.